concepticondata/conceptlists/Verheyen-2019-1000.tsv

```diff
@@ -0,0 +1,1001 @@
+Verheyen-2019-1000-60	60	beige	beige	BEIGE	3707	7.8	2.35	5.9	2.75	3.3	6.55	3.4	5	2	37743	1	191	73	0.700583079428269	0.000139002405622
+Verheyen-2019-1000-61	61	bejaard	elderly	OLD (AGED)	2112	7.4	2	5.3	3	4.4	6.75	3.85	7	2	40344	4	288	84	1.01741566986598	0.000113876862515
+Verheyen-2019-1000-62	62	beknopt	short	SHORT	1645	10.75	3.8	4.45	3.2	4.05	3.95	4	7	2	23060	0	238	17	0.572122035398519	5.01697316509991E-05
+Verheyen-2019-1000-63	63	bekommerd	concerned			10.1	4.55	3.65	3.68421052631579	3.1	4.9	4.1	9	3	77625	0	121	1	0.150611538270482	1.30037132825707E-05
+Verheyen-2019-1000-64	64	bekrompen	narrow	NARROW	1267	10.4210526315789	2.95	3.31578947368421	2.7	3.3	4.25	2.4	9	3	73434	1	101	133	1.09499734727642	0.000240471070252
+Verheyen-2019-1000-65	65	bekwaam	competent			10.2	4.2	3.65	5	4.05	3.6	4.85	7	2	38847	0	626	182	1.26198302030664	0.000117059453003
+Verheyen-2019-1000-66	66	belachelijk	ridiculous			7.7	4.6	3.7	3.6	5.95	3.45	2.5	11	4	110190	0	1448	2838	1.8226941043475	0.000296387688204
+Verheyen-2019-1000-67	67	belangrijk	important	IMPORTANT	1285	5.95	5.2	4.45	5.4	6.05	4.6	5.1	10	3	95759	0	18171	12156	10.5542371929662	0.001966238897868
+Verheyen-2019-1000-68	68	belangrijker	upper			6.4	5.1	3.95	5.25	5.45	4.2	5.2	12	4	144498	0	18171	45	NaN	NaN
+Verheyen-2019-1000-69	69	belangrijkst	chief			7.3	5.45	5.1	6.1	4.9	3.25	5.25	12	3	116119	0	18171	872	NaN	NaN
+Verheyen-2019-1000-70	70	belangstellend	interested			11.8421052631579	5.05263157894737	4.45	4.5	3.3	3.15	5.25	14	4	186917	0	119	2	0.03714709205279	2.66842965257957E-06
+Verheyen-2019-1000-71	71	beminnelijk	amiable			10.05	4.55555555555556	4	4.61111111111111	3.55	3.95	5.75	11	4	89318	0	216	35	0.102395738038682	8.38602831060677E-06
+Verheyen-2019-1000-72	72	benauwd	muggy	MUGGY	3757	9.6	4.2	4.1	3.6	4.5	5.05	2.6	7	2	50086	0	350	105	1.21847208778613	0.000170076594344
+Verheyen-2019-1000-73	73	bepaald	specific			9.15	3.6	3.5	4.05	4.95	4.3	4.6	7	2	40622	1	9389	1016	0.375389274370291	6.15496078645E-05
+Verheyen-2019-1000-74	74	beperkt	limited			9.45	3	4.3	3.1	5.1	2.95	3.05	7	2	63368	0	2357	270	2.16433293965715	0.000548804862426
+Verheyen-2019-1000-75	75	bergachtig	mountainous	MOUNTAINOUS	3755	7.8	4.8	5.65	4.8	2.7	6.6	3.6	10	3	96827	0	85	8	0.092028559513635	1.37066167032502E-05
+Verheyen-2019-1000-76	76	beroemd	famous			5.7	5.45	5	5.35	5.25	5.85	4.6	7	2	71959	2	1353	2007	2.31968236829124	0.000291717936312
+Verheyen-2019-1000-77	77	beschikbaar	available			8.75	4.25	4.4	3.95	5.05	3.4	4.55	11	3	74481	0	2035	723	0.152208672323134	4.53730666413621E-05
+Verheyen-2019-1000-78	78	besmeurd	black			9.52631578947369	4.52631578947368	4.75	3.75	2.9	5.2	2.5	8	2	38696	0	0	0	0.029744401808838	4.34758782420281E-06
+Verheyen-2019-1000-79	79	best	best			5.2	5.55	4.1	5.8	5.6	3.5	5.95	4	1	34613	16	2885	13066	1.04597302363008	0.000322541552982
+Verheyen-2019-1000-80	80	bestaand	living	BE ALIVE	1422	8.4	3.25	4.45	3.95	4.6	2.4	4.55	8	2	81014	0	1951	1	0.100759975374276	2.05143664998312E-05
```

Comment:
bestaand means existing, not just necessarily for living things

Reply:
BE 1579 ?